appium/webdriver/extensions/android/common.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from selenium import webdriver
+
+from appium.webdriver.mobilecommand import MobileCommand as Command
+
+
+class Common(webdriver.Remote):
```

Comment:
I would rather keep these commands in the main driver class

Reply:
I've had a chat with @KazuCocoa on this and added per his recommendation.

Reply:
I mean the term "Common" does not properly describe the purpose of all these commands, because, for example the `open_notifications` method only works for Android

Reply:
I've exactly asked the same question during my chat with him - And that's why this common.py is under /extensions/android class

Reply:
yeah, I thought only for Android was much better to put them in _android_ module than the main driver for all